### PR TITLE
Features/rationalize tests

### PR DESCRIPTION
**Pull request by thypon**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -203,11 +183,21 @@ runs:
                 region: options.region
               })
 
+              let watermark = patch.watermark
+
+              if (debug) {
+                watermark = options.bedrock_aws_iam_role_arn
+                              ? `bedrock debug - ${watermark}`
+                              : options.anthropic_api_key
```
Comment (diracdeltas) [inline action.yml:191]: why are we logging the anthropic api key?

Comment (diracdeltas) [reply action.yml:191]: noting that it's also logged in https://github.com/brave/pull-merge/pull/119/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R152; should this be redacted from debug output?

Comment (thypon) [reply action.yml:191]: that's not the case. `? :` it's a neat operator you can use to chain more than one equality/existence check.

that can be translated in:

```
if (options.bedrock_aws_iam_role_arn) {
  watermark = `bedrock debug - ${watermark}`
} else if (options.anthropic_api_key) {
  watermark = `anthropic debug - ${watermark}`
} else {
  watermark = `openai debug - ${watermark}`
}
```

Comment (thypon) [reply action.yml:191]: anything that is a secret is automatically redacted by github output, btw, with asterisks.
The only way to output that is first encoding, then output.

**Review by diracdeltas — APPROVED** [review]

**Pull request merged**